run/archiveloop

```diff
@@ -247,6 +247,8 @@ function disconnect_usb_drives_from_host () {
 function archive_teslacam_clips () {
   log "Starting recording archiving..."
 
+  /root/bin/send-push-message "TeslaUSB:" "Archiving started at $(date)"
```

Comment:
won't it be confusing if it always sends this message, but only sends the completion message when it actually archived something?

Reply:
is it possible for the 'starting archiving' message specify if the prior archiving process succeeded or failed? That might alleviate the concern above

Reply:
@marcone - Fair point. I think the message can be changed to something like "Reconnected to WiFi. New files (if any) will be synced." I am not a native english speaker so you can let me know if there is something that's more succinct. 

@hoopsta1423 - I think that's a good suggestion, but I will need to think about it. I think that can be another push.

Reply:
Another option might be to do a quick scan of /mnt/cam/TeslaCam/SavedClips and only send the start message if so. I'll merge this for now so people can try it out and give feedback.